src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -110,6 +115,25 @@ public void configure(Map<String, ?> configs) {
     private class StorageChecker implements Runnable {
         private final Thread storageCheckerThread = new Thread(this, "storage-quota-checker");
         private volatile boolean running = false;
+        private final Metric kafkaBrokerAllUsedBytes = Metrics.newGauge(mNameCreate("AllUsedBytes"), new Gauge<Long>() {
```

Comment:
Not convinced that this name is especially clear, what about

```suggestion
        private final Metric kafkaBrokerAllUsedBytes = Metrics.newGauge(mNameCreate("TotalStorageUsedBytes"), new Gauge<Long>() {
```

Reply:
Works for me!